lumispy/tests/signals/test_cl_spectrum.py

```diff
@@ -48,11 +48,11 @@ def test__make_signal_mask(self):
 
     def test_remove_spikes(self):
         s = CLSpectrum(np.ones((2, 3, 30)))
-        np.random.seed(1)
+        np.random.seed(np.random.randint(666666))
```

Comment:
If I understand correctly, `np.random.randint` is still using the legacy random generator: https://numpy.org/doc/stable/reference/random/generated/numpy.random.randint.html. Would it make sense to use the new generator?
The legacy generator is not deprecated but it will most likely be at some point.

Reply:
For what I understood the np.random.seed statement has no effect as the following add_gaussian_noise from hyperspy does not take it into account and applies its own seed using the generator object.

I updated the tests to remove this line